tcms/rpc/tests/test_testcase.py

```diff
@@ -569,14 +697,19 @@ def verify_api_with_permission(self):
         self.assertEqual(result[str(self.case_2.pk)], 15)
         self.assertEqual(result[str(self.case_3.pk)], 25)
 
+        # Test query None
+        result = self.rpc_client.TestCase.sortkeys(None)
```

Comment:
You'd better call the method as `TestCase.sortkeys()` which makes sure to also use the specified default value in the method signature. Same applies for `TestCase.filter()` method.